packages/neos-ui-redux-store/src/UI/Inspector/index.ts

```diff
@@ -61,6 +62,10 @@ const apply = () => createAction(actionTypes.APPLY);
 const discard = (focusedNodeContextPath: NodeContextPath) => createAction(actionTypes.DISCARD, {focusedNodeContextPath});
 const escape = () => createAction(actionTypes.ESCAPE);
 const resume = () => createAction(actionTypes.RESUME);
+const selectElement = (selectedElementContextPath: NodeContextPath) =>
```

Comment:
I would name it: `selectedElement` to keep it consistent with `selectedElementContextPath`.

Reply:
It is an action creator though, so it describes the *action* of selecting an element - not the result. `selectElement` is imho consistent with the the other action creators above (`discard`, `escape`, `resume` - these are all imperatives).

Reply:
Okay. Understood! 👍🏽